index.js

```diff
@@ -29,9 +30,44 @@ function sendBootStatus( status ) {
 	process.send( { boot: status } );
 }
 
-console.log( chalk.yellow( '%s booted in %dms - http://%s:%s' ), pkg.name, ( Date.now() ) - start, host, port );
+console.log(
+	chalk.yellow( '%s booted in %dms - %s://%s:%s' ),
+	pkg.name,
+	( Date.now() ) - start,
+	protocol,
+	host,
+	port
+);
 
-server = http.createServer( app );
+// Start a development HTTPS server.
+if ( protocol === 'https' ) {
+	const { execSync } = require( 'child_process' );
+	const execOptions = { encoding: 'utf-8', windowsHide: true };
+	let key = './config/server/key.pem';
+	let certificate = './config/server/certificate.pem';
+
+	if ( ! fs.existsSync( './config/server/key.pem' ) || ! fs.existsSync( './config/server/certificate.pem' ) ) {
```

Comment:
nit: you can use the `key` and `certificate` variables here.

Reply:
Done.